pyproject.toml

```diff
@@ -62,6 +62,7 @@ dependencies = [
     'typing-extensions>=4.5.0',
     'annotated-types>=0.4.0',
     "pydantic-core==0.33.0",
+    "hypothesis>=6.75.3",
```

Comment:
I don't think this is a dependency at all - if Hypothesis is not already imported, then it has no effect; and it's beautifully narrow in scope so that there's no downside if not.